app/views/best.html

```diff
@@ -40,7 +40,7 @@
               <span ng-if="item.normalStats[$index].scaled > 0"><small ng-bind="item.normalStats[$index].scaled"></small>/<small ng-bind="item.normalStats[$index].split"></small></span>
             </div>
           </div>
-          <div dim-store-item item-data="item" store-data="normal.getStore(item.owner)"></div>
+          <div dim-simple-item item-data="item"></div>
```

Comment:
Oh, sure. It hadn't crossed my mind that you could click on these and use the move popup - I was just thinking that it was overkill (and weird that you can drag them).